sql/2020/21_Resource_Hints/21_12a.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+# 21_12a: Count of pages using native lazy loading
+SELECT
+  countif(REGEXP_CONTAINS(body, r'(?im)<(?:source|img)[^>]*loading=')) as lazyCount,
```

Comment:
Use the `almanac.images.loading_values` custom metric to avoid querying the very expensive `response_bodies` table.

See https://github.com/HTTPArchive/legacy.httparchive.org/blob/cfe880c6ca3bf98433a5fe2103acd3929c518b24/custom_metrics/almanac.js#L513